cogboard-app/src/main/resources/initData/endpoints.json

```diff
@@ -1,14 +1,14 @@
 {
   "endpoints": [
     {
-      "id": "api-mocks-endpoint",
+      "id": "endpoint1",
```

Comment:
Okay, explain this; we had a nice, descriptive name, and now we don't.

Reply:
because of method: